### ISSUE TITLE
Not working in fresh instalation?

### ISSUE THREAD

**Issue by n-aspen** [mathieudutour/timezone-butler#98]
**Describe the bug**
Timezone Butler doesn't seem to be working after fresh installation. 

**To Reproduce**
1. Send a time `12:26pm` as part of a message 
2. Have another user in another timezone check for Timezone Butler responses 
3. Find nothing 

**Expected behavior**
For Timezone Butler to provide a converted time to users in different timezones. 

**Screenshots**
![image](https://github.com/user-attachments/assets/1bf54730-324f-4df8-a8ca-f016dea25f4b)
![image](https://github.com/user-attachments/assets/848547e2-7ede-4d92-bffb-246e81f2d89a)
![image](https://github.com/user-attachments/assets/8cfb35cb-fff6-461e-a0d6-2a72d71b8dce)

**Additional context**
Fresh installation, we weren't able to get any of the features to work at any point. Because of the lack of documentation, it's unclear if we're missing something or if it's just not working as intended. 

Timezone Butler is in the channel, confirmed in the Slack UI under "integrations" and the Slack App Directory.

**Comment by ywwg** [comment]
not working for us either

**Comment by mathieudutour** [comment]
Yeah Slack killed it and wants me to do a security pen testing and a bunch of stuff which I can't be bothered with. Feel free to host it yourself and it should work fine as long as you don't try to publish it on the slack store

**Comment by sheam** [comment]
Ahh that is too bad, this would have been perfect for us. Cheers.
